__tests__/index.test.js

```diff
@@ -1,9 +1,47 @@
-// You can import your modules
-// const index = require('../index')
+const bot = require('..')
 
-test('that we can run tests', () => {
-  // your real tests go here
-  expect(1 + 2 + 3).toBe(6)
+const { createRobot } = require('probot')
+
+describe('your-bot', () => {
+  let robot
+  let github
+
+  beforeEach(() => {
+    robot = createRobot()
+    bot(robot)
+
+    github = {
+      issues: {
+        create: jest.fn(() => Promise.resolve())
+      },
+      repos: {
+        createStatus: jest.fn(() => Promise.resolve()),
+        getContent: jest.fn(() => Promise.resolve({data: {content: Buffer.from('tv: law and order').toString('base64')}}))
+      }
+    }
```

Comment:
I know this is the recommended way of testing in the docs still, but I'd actually like to replace this with `nock`. See https://github.com/probot/probot/blob/master/test/plugins/stats.test.js for an example of an app being tested with nock.